lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -132,9 +130,11 @@ class EffectiveClangContext {
   /// Retrieve the unresolved context name.
   StringRef getUnresolvedName() const {
     assert(getKind() == UnresolvedContext);
-    return StringRef(Unresolved.Data, Unresolved.Length);
+    return StringRef(Unresolved.Data, UnresolvedLength);
   }
 };
+static_assert(sizeof(EffectiveClangContext) <= 2 * sizeof(void *),
```

Comment:
Yes, people have used the compiler on 32-bit Linuxes. We could decide not to care about that, but we should do it deliberately and discuss it first.